pkgs/modules/python/default.nix

```diff
@@ -100,6 +99,14 @@ let
 
   sitecustomize = pkgs.callPackage ./sitecustomize.nix { };
 
+  sitecustomized-python = pkgs.runCommand "python-replit-sitecustomized" {} ''
+    ls -al ${python.outPath}
+    ${pkgs.rsync}/bin/rsync -av --chown=$USER --chmod=+w ${python.outPath}/ $out/
```

Comment:
I am baffled that this is the only way. Doesn't this double the disk required for all python packages?

Reply:
ooooh i think you're right, i don't actually force this python's usage anywhere. i'll fix that in the morning